src/modules/SX127x/SX1278.h

```diff
@@ -248,9 +248,11 @@ class SX1278: public SX127x {
     /*!
       \brief Gets recorded signal strength indicator of the latest received packet for LoRa modem, or current RSSI level for FSK modem.
 
+      \param skip_activation in OOK/FSK mode this function will put receiver un receive mode and in standby. Make it TRUE if you don't want this behaviour.
+
       \returns Last packet RSSI for LoRa modem, or current RSSI level for FSK modem.
     */
-    float getRSSI();
+    float getRSSI(bool skip_activation=false);
```

Comment:
Please reflect the changes made in SX1278.h/SX1278.cpp in SX1272.h/SX1272.cpp too.

Reply:
done